qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -97,7 +98,11 @@ def properties(self, refresh=False, datetime_filter=None):
 
         if refresh or self._properties is None:
             api_properties = self._api.backend_properties(self.name(), extra_filter=api_filter)
-            self._properties = BackendProperties.from_dict(api_properties)
+            # Todo: from_dict fails because it could map an empty dictionary to the BackendProperties Schema
+            try:
+                self._properties = BackendProperties.from_dict(api_properties)
+            except ModelValidationError as ex:
+                self._properties = ex
```

Comment:
When there are no configurations for a backend, api_properties is set to an empty dictionary. When trying to map the empty dictionary to a BackendProperties schema, a ModelValidationError is thrown with the following message:

{'qubits': ['Missing data for required field.'], 'general': {0: {'value': ['Missing data for required field.'], 'date': ['Missing data for required field.'], 'unit': ['Missing data for required field.'], 'name': ['Missing data for required field.']}}, 'gates': {0: {'qubits': ['Missing data for required field.'], 'parameters': {0: {'value': ['Missing data for required field.'], 'date': ['Missing data for required field.'], 'unit': ['Missing data for required field.'], 'name': ['Missing data for required field.']}}, 'gate': ['Missing data for required field.']}}, 'backend_version': ['Missing data for required field.'], 'last_update_date': ['Missing data for required field.'], 'backend_name': ['Missing data for required field.']}

As of right now, I'm returning the exception message above. Is there another message that is preferable to return, or is this message okay to return?